lms/djangoapps/branding/api.py

```diff
@@ -564,27 +579,72 @@ def get_base_url(is_secure):
 
 def get_logo_url(is_secure=True):
     """
-    Return the url for the branded logo image to be used
+    Return the url for the branded logo image to be used.
+
+    Preference of the logo should be,
+        Look for site configuration override and return absolute url
+        Absolute url of brand Logo if defined in django settings
+        Relative default local image path
+
     Arguments:
         is_secure (bool): If true, use HTTPS as the protocol.
     """
+    brand_logo_url = settings.LOGO_URL
+    default_local_path = 'images/logo.png'
+    logo_url_from_site_config = configuration_helpers.get_value('logo_image_url')
+    university = configuration_helpers.get_value('university')
```

Comment:
What's the university here, out of curiosity?

Reply:
in prod we do not have the value set so I assume some openedX users would be using that.